modules/ossm-cr-tracing-jaeger.adoc

```diff
@@ -16,7 +16,6 @@ This example illustrates tracing parameters for the {ProductName} custom resourc
       jaeger:
```

Comment:
Indentation is off. Should be two spaces throughout.

Reply:
@luksa This entire page was rewritten and is now under ossm-configuring-jaeger.adoc.